deployments/dockerfiles/gosec/Dockerfile

```diff
@@ -1,11 +1,18 @@
 # Dockerfile used to create "husyci/gosec" image
 # https://hub.docker.com/r/huskyci/gosec/
 
-FROM golang:alpine 
+FROM golang:alpine as builder
 
 RUN apk update && apk upgrade \
 	&& apk add --no-cache alpine-sdk git \
 	&& apk add --update --no-cache git openssh-client jq
```

Comment:
Hello, @renatoaquino 

The new image works, but we are missing the 2 packages that were built at these lines that are required by huskyCI (git and jq).

```
$ docker run -it huskyci/newgosec sh
/go # git clone https://github.com/globocom/huskyCI
sh: git: not found
/go #
```

Reply:
Nice, @henriquebonadio! Good catch!

Reply:
Since I don't know what's the huskyCI workflow, I've assumed that git was needed only by the `go get` command.
The fix is in place, but now the container size increased:
```
gosec          latest              fc316f0abe32        About a minute ago   31.1MB
```

XD